bears/general/LineCountBear.py

```diff
@@ -39,3 +50,12 @@ def run(self, filename, file, max_lines_per_file: int,
                                  extra=file_length-max_lines_per_file)),
                 severity=RESULT_SEVERITY.NORMAL,
                 file=filename)
+
+        elif file_length < min_lines_per_file:
+            yield Result.from_values(
+                origin=self,
+                message=('This file has {} lines, while {} lines are '
+                         'required.'
+                         .format(file_length,
+                                 min_lines_per_file - file_length)),
```

Comment:
I think the message could be rephrased to:

> This file has `file_length` lines, while `min_lines_per_file` lines are required.

The phrase "n lines are required" specifies the minimum number of lines required, not the number of lines to be added to meet the requirement.

Reply:
@HizkiFW Thanks, I forgot about change this :) Corrected :)